Tasks/BashV3/bash.ts

```diff
@@ -128,6 +128,11 @@ async function run() {
             ignoreReturnCode: true
         };
 
+        process.on("SIGINT", () => {
+            console.log('Started cancellation of executing script');
```

Comment:
Thanks for adding log messages, but could you please use ` tl.debug` for output instead of ` console.log`?